scripts/Items/Item.cs

```diff
@@ -0,0 +1,41 @@
+using Godot;
+using Godot.Collections;
+
+[GlobalClass]
+[Tool]
+public partial class Item : Resource
+{
+	[ExportGroup("Item properties")]
+	[Export] public string itemName { set; get; }
+	[Export(PropertyHint.Enum, "Common,Rare,Legendary")]
+
+	public string itemGrade { set; get; }
+	
+	[ExportGroup("Item visuals")]
+	[Export] public Texture itemImage { set; get; }
+	[Export] public PackedScene itemModel { set; get; }
```

Comment:
Perhaps a private set? Since they are initialized in the inspector.

Reply:
fix fixed